File: data/conala-train_code-davinci-002_1666524963/Java/480.java
List<Integer> list1 = Arrays.asList(1, 2, 3, 4, 5);
List<Integer> list2 = Arrays.asList(3, 4, 5, 6, 7);

list1.stream()
    .filter(i -> !list2.contains(i))
    .collect(Collectors.toList());

File: data/conala-train_code-davinci-002_1666524963/Python/480.py
def find_non_common_elements(list1, list2):
    return list(set(list1) - set(list2))

list1 = [1, 2, 3, 4, 5]
list2 = [5, 6, 7, 8, 9]

print(find_non_common_elements(list1, list2))